platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionManagePageAddDeleteTestIT.java

```diff
@@ -0,0 +1,232 @@
+package org.exoplatform.platform.qa.ui.platform.wiki;
+
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.wiki.WikiLocators.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.codeborne.selenide.Condition;
+
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.RichTextEditor;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiDraftPage;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiHomePage;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiManagement;
+
+@Tag("wiki")
+@Tag("smoke")
+
+public class WikiBasicActionManagePageAddDeleteTestIT extends Base {
+
+  HomePagePlatform  homePagePlatform;
+
+  WikiHomePage      wikiHomePage;
+
+  ManageLogInOut    manageLogInOut;
+
+  WikiManagement    wikiManagement;
+
+  NavigationToolbar navigationToolbar;
+
+  RichTextEditor    richTextEditor;
+
+  WikiDraftPage     wikidraftpage;
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+    info("Start setUpBeforeMethod");
+    homePagePlatform = new HomePagePlatform(this);
+    wikiHomePage = new WikiHomePage(this);
+    manageLogInOut = new ManageLogInOut(this);
+    wikiManagement = new WikiManagement(this);
+    navigationToolbar = new NavigationToolbar(this);
+
+    wikidraftpage = new WikiDraftPage(this);
+    try {
+      richTextEditor = new RichTextEditor(this);
```

Comment:
Why is this object creation catched ? If it fails, it will be null, and since we only display a stacktrace in the catch, it will continue, so I guess some tests will throw NullPointerException.
We should probably fail directly if this object cannot be instantiated (so no try/catch).
WDYT ?